clog/builder.go

```diff
@@ -49,34 +54,49 @@ func newBuilder(ctx context.Context) *builder {
 // we can chase later.  This is all still in the early/poc stage and needs
 // additional polish to shine.
 func (b builder) log(l logLevel, msg string) {
-	cv := clues.In(b.ctx).Map()
-	zsl := b.zsl
+	var (
+		cluesNode = clues.In(b.ctx)
+		cv        = cluesNode.Map()
+		zsl       = b.zsl
+	)
+
+	// set up an otel logging record
+	// if otelLog is nil, this will eventually no-op
+	record := log.Record{}
```

Comment:
Related to duplicate imports, we should use `otellog.Record` instead of `log.Record` here.